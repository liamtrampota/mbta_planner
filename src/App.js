import MBTAJourney from './components/MBTAJourney';

function App() {
  return (
    <div>
      <MBTAJourney />
    </div>
  );
}

export default App;
