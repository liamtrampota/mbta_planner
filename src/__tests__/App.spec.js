import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

test('renders App', () => {
  const component = shallow(<App/>);
  expect(component).toMatchSnapshot();
});
