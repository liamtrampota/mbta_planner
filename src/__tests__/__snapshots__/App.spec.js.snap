// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders App 1`] = `ShallowWrapper {}`;
